hack/test-integration.sh

```diff
@@ -33,7 +33,16 @@ if [ "$COVER" -eq 1 ]; then
 fi
 
 # Run the tests and collate the results.
-bats -t $ROOT/test/*.bats
+TESTS=""
+for item in ${BATS[@]};
```

Comment:
Sure, we could entirely ignore this and just assume we have relative paths to the project's root. I'll update the commits.